custom_components/vesync/sensor.py

```diff
@@ -201,6 +198,22 @@ def name(self):
         """Return sensor name."""
         return f"{super().name} air quality"
 
+    @property
+    def device_class(self):
+        """Return the air quality device class."""
+        return SensorDeviceClass.ENUM
+    
+    @property
+    def options(self):
+        """Return the air quality options."""
+        # Note that while the manual (https://cdn.accentuate.io/4847302017112/1617079386388/02.00_M1_LV-H131-RWH-Grayscale_2021-03-02_US_en-revised.pdf?v=0) specifies the top air quality as "very good" the API actually reports "Excellent".
+        return ["Excellent", "Good", "Moderate", "Bad"]
```

Comment:
Not sure options should be hardcoded. Can you have a look at the diagnostics of your device? It usually lists the options for this kind of variable.